Marlin/ultralcd.cpp

```diff
@@ -2118,18 +2118,22 @@ void kill_screen(const char* lcd_msg) {
      * UBL Load Mesh Command
      */
     void _lcd_ubl_load_mesh_cmd() {
-      char UBL_LCD_GCODE[8];
+      char UBL_LCD_GCODE[25];
       sprintf_P(UBL_LCD_GCODE, PSTR("G29 L%i"), ubl_storage_slot);
       enqueue_and_echo_command(UBL_LCD_GCODE);
+      sprintf_P(UBL_LCD_GCODE, PSTR("M117 Map %i loaded."), ubl_storage_slot);
```

Comment:
Also `Map %i loaded` and `Map %i saved` should be translatable strings in the language files. Thus:
```cpp
sprintf_P(UBL_LCD_GCODE, PSTR("M117 " MSG_MAP_LOADED "."), ubl_storage_slot);
```